### PR TITLE
Improve examples for 'max server memory (MB)'

### PR DESCRIPTION
**Pull request by jzabroski**
Just giving some helpful tips I find useful.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@jzabroski : Thanks for your contribution! The author, @MikeRayMSFT, has been notified to review your proposed change.

File: docs/database-engine/configure-windows/server-memory-server-configuration-options.md [hunk 1]
```diff
@@ -118,26 +118,29 @@ To enable the lock pages in memory option:
...
-GO  
-RECONFIGURE;  
-GO  
-```  
-  
+
+## Examples
+
+### Example A. Set the max server memory option to 4 GB.
+ The following example sets the `max server memory` option to 4 GB.  Note that although `'show advanced options'` specifies the name of the option as `max server memory (MB)`, the example demonstrates omitting the `(MB)`.
```
Comment (MikeRayMSFT) [inline docs/database-engine/configure-windows/server-memory-server-configuration-options.md]: Please change: 
> 'show advanced options' specifies 

to

> `sp_configure` displays

Comment (MikeRayMSFT) [reply docs/database-engine/configure-windows/server-memory-server-configuration-options.md]: @jzabroski - if you can make these changes, we will be able to update the article.

Thank you for your submission.

**Comment by jzabroski** [comment]
Updated. First time updating a PR from my phone so forgive me if it didn't update.

File: docs/database-engine/configure-windows/server-memory-server-configuration-options.md (hunk 1 above)
Comment (jzabroski) [reply docs/database-engine/configure-windows/server-memory-server-configuration-options.md]: @MikeRayMSFT  Sorry, FastHub on android doesnt seem to have a Resolve Conversation button. I fixed this last night.

**Comment by MikeRayMSFT** [comment]
Thank you @jzabroski 
#sign-off

**Pull request merged**